### DISCUSSION TITLE
Filter option '- *' not working. it prevents all other rules as well

### DISCUSSION THREAD

**Discussion by MohsinQK** [sourcebroker/deployer-extended-media#5] [General]
Filter option '- *' not working. it prevents all other rules as well
Copied from Readme file

```
set('media',
    [
     'filter' => [
         '+ /fileadmin/',
         '- /fileadmin/_processed_/*',
         '+ /fileadmin/**',
         '+ /uploads/',
         '+ /uploads/**',
         '- *'
    ]
]);
```

https://stackoverflow.com/a/46227148/10422426

**Comment by kszymukowicz** [comment]
It looks ok.
How do you installed TYPO3? 
Is there "public" folder ?

**Comment by MohsinQK** [comment]
@kszymukowicz 
yes it's a composer installation. with a public folder.
and I have set the paths with /public/  prefixed with all the folders above.
but it only works removing - *  but it syncs everything. like vendor etc.

**Comment by kszymukowicz** [comment]
So if its in public folder then tell me your final config for set('media',

**Comment by MohsinQK** [comment]
@kszymukowicz 
This is my final config for set('media')

```
set('media',
    [
     'filter' => [
         '+ /public/fileadmin/',
         '- /public/fileadmin/_processed_/*',
         '+ /public/fileadmin/**',
         '+ /public/uploads/',
         '+ /public/uploads/**',
         '- *'
    ]
]);
```

**Comment by kszymukowicz** [comment]
You miss this line
https://github.com/sourcebroker/deployer-extended-typo3/blob/9192bb8a0e10071b38b1edffaee7e3b54c7f9487/deployer/default/set.php#L51

**Comment by kszymukowicz** [comment]
@MohsinQK 
so is it working ? would be good to close it
